lib/Widgets/OutgoingTransmissionListEntry.vala

```diff
@@ -46,10 +47,14 @@ public class Drop.Widgets.OutgoingTransmissionListEntry : TransmissionListEntry
     }
 
     private void build_ui () {
-        cancel_button = new Gtk.Button.from_icon_name ("process-stop-symbolic", Gtk.IconSize.BUTTON);
+        cancel_button = new Gtk.Button.with_label (_("Cancel"));
         cancel_button.valign = Gtk.Align.CENTER;
         
+        stop_button = new Gtk.Button.from_icon_name ("process-stop-symbolic", Gtk.IconSize.SMALL_TOOLBAR);
+        stop_button.valign = Gtk.Align.CENTER;
+
         action_area.add (cancel_button);
+        action_area.show_all ();
```

Comment:
but here it looks like it does work without `show_all`: https://github.com/Drop-Project/Vala-Server/blob/UI-update/lib/Widgets/IncomingTransmissionListEntry.vala#L211